shell/platform/darwin/ios/rendering_api_selection.h

```diff
@@ -17,11 +17,16 @@ enum class IOSRenderingAPI {
   kMetal,
 };
 
-IOSRenderingAPI GetRenderingAPIForProcess();
+IOSRenderingAPI GetRenderingAPIForProcess(bool force_software);
 
-Class GetCoreAnimationLayerClassForRenderingAPI(
-    IOSRenderingAPI rendering_api = GetRenderingAPIForProcess());
+Class GetCoreAnimationLayerClassForRenderingAPI(IOSRenderingAPI rendering_api);
 
 }  // namespace flutter
 
+#if TARGET_OS_SIMULATOR
+#define METAL_IOS_VERSION_BASELINE 13.0
```

Comment:
Please document how these baselines were selected.

Reply:
This was added by @kavantix - Adding what I assume is correct as 

```cpp
// Metal support was added for simulators in iOS 13.
// Otherwise, the lowest supported version in Skia is iOS 10.
```

Reply:
Ahh I see, I'm moving the old comment from the impl file to here, and adding another line about when sim support was added.